SocialFilter/sections/general_elemhide.txt

```diff
@@ -702,7 +702,7 @@ designboom.com,livescience.com,pbtech.co.nz,peoplesworld.org,space.com##.fa-face
 ##.share-article-tools
 ##.share-article-vertical
 ##.share-bar-desktop
-~upi.com##.share-block
+~last.fm,~upi.com##.share-block
```

Comment:
I don't see the element even without AdGuard.
According to the screenshot, it's share.

Reply:
It required an account. I forgot to note that. sorry.
It is not just a copy of the URL. But sharing may be sharing.
Apologies.

Reply:
I've also checked it with account and don't have it.
Anyway, it can be count as share and you removed some similar element in your pr's :)